include/internal/catch_run_context.cpp

```diff
@@ -67,6 +67,23 @@ namespace Catch {
         GeneratorTracker::~GeneratorTracker() {}
     }
 
+    class RedirectedStreams {
```

Comment:
this class semantically belongs to `include/internal/catch_stream.h` with implementation inside `include/internal/catch_stream.cpp`

Reply:
Can I move the class to catch_output_redirect.h/.cpp instead?

I need to reference the `RedirectedStdOut `and `RedirectedStdErr` classes from the header file (unfortunately I cannot forward declare them), and these two classes are defined in catch_output_redirect.h (which already #includes catch_stream.h), so I want to avoid the circular include dependency. Plus the class is not really a stream, more a form of output redirection. I could rename the class to something like `RedirectOutputToStrings` if it's clearer.